changelog/next/features/3883.md

```diff
@@ -0,0 +1,3 @@
+Managed pipelines now contain a new `total_runs` parameter that counts all
+started runs. This counter also serves as the new `run_id` for the
```

Comment:
Well, our current pipeline IDs are numbers, too...
Internally the `run_id` for diagnostics and metrics is a string, just like `pipeline_id` - I can change the name, but then we'd have a naming misalignment between `pipeline_id` and `run`. IDK if that is a big problem though.

Reply:
I think `pipeline_id` should stay a string. That it always contains a number happened more by accident than by design. At some point in the future, we should just use something like `fmt::to_string(uuid::random())` for the id when we create a new pipeline.

`run`/`run_id` should be a number, as otherwise the user cannot perform arithmetic operations with it—which they're supposed to do if they want to compare it to `total_runs` for the same pipeline.

Reply:
I implemented `run` as a number now.